### DISCUSSION TITLE
Feature to show when a claim was created

### DISCUSSION THREAD

**Discussion by Snorry317** [GriefPrevention/GriefPrevention#1756] [Ideas]
Would it be possible to implement a feature that tells Admins when a claim was created?

**Comment by Jikoo** [comment]
Do you mean a broadcast or an addition of the claim creation date to inspection? An addon could definitely do either. The abridged logs already contain claim creation for manual review.

**Comment by SiuanSong** [comment]
I would like to see this added on in the form of inspecting claims, especially since claims aren't stored by player, it makes it pretty much impossible to see when a claim was created.  We usually have to rely on CoreProtect logs, but we don't keep those around forever. I have never seen anything in the abridged logs about claim creation.

**Comment by BLU3NINJ4** [comment]
Sorry i know this is from some time ago But i really like to know if there is any way to check the creation date of claims with this plugin
Trying to use coreprotect isnt really a valid way since it only shows when stuff was built for example but the claim might be older

**Comment by snurre0** [comment]
I came on here to suggest exactly this, and I found this discussion.

When right clicking with a stick inside a claim, a nice addition would be the claim creation date in the same output.
Adding created timestamp to each claim file, and possibly also modified date, and then displaying this info when using claim visualization, would be a nice addition!
Probably also controlling this information a similar way as the "Last Login" info is controlled, so not everyone can see this.

**Comment by ThePro1604** [comment]
this will be such a handy tool for moderators and admins on servers!
